pkg/sql/catalog/catpb/privilege.go

```diff
@@ -40,6 +40,11 @@ const (
 	// These descriptors should have all the correct privileges and the owner field
 	// explicitly set. These descriptors should be strictly validated.
 	Version21_2
+
+	// Version23_2 corresponds to descriptors created in 23.2 and onwards. These
+	// descriptors that are function descriptions should have EXECUTE privileges
+	// for the public role.
+	Version23_2
```

Comment:
Does this require a new cluster version? I'm seeing `TestSystemDatabaseSchemaBootstrapVersionBumped` fail with:

```
Check whether you need to bump systemschema.SystemDatabaseSchemaBootstrapVersion
        and then update prevSystemHash to "7d089e4c69d8d6b164b3d1a04e0f3ad5f487f9138c9585bd00b37f397691a090".
        The current value of SystemDatabaseSchemaBootstrapVersion is 1000023.1-32.
```